packages/manager/modules/at-internet-configuration/package.json

```diff
@@ -11,11 +11,12 @@
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
   "main": "./src/index.js",
-  "dependencies": {},
   "peerDependencies": {
+    "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
     "@ovh-ux/manager-core": "^12.0.0 || ^13.0.0",
-    "@ovh-ux/ng-at-internet": "^5.7.0",
+    "@ovh-ux/ng-at-internet": "^5.8.1",
     "@ovh-ux/ng-at-internet-ui-router-plugin": "^3.1.2",
+    "@ovh-ux/ng-shell-tracking": "^0.0.0",
```

Comment:
**question:** To meet to first release of `@ovh-ux/ng-shell-tracking` component, shouldn't be this?
```suggestion
    "@ovh-ux/ng-shell-tracking": "^0.0.0 || ^1.0.0",
```